src/main/java/dk/cngroup/lentils/controller/client/CypherController.java

```diff
@@ -107,56 +107,61 @@ public String cypherDetail(@PathVariable("id") final Long id,
         if (status.equals(CypherStatus.LOCKED)) {
             throw new ResourceNotFoundException("locked cypher", id);
         }
-        CodewordFormDTO codewordFormDto = new CodewordFormDTO();
-        setDetailModelAttributes(model, user, cypher, status, codewordFormDto);
+        ClientVerifyCodewordActionDTO clientVerifyCodewordActionDto = new ClientVerifyCodewordActionDTO();
+        clientVerifyCodewordActionDto.setCypherId(id);
+        setDetailModelAttributes(model, user, cypher, status, clientVerifyCodewordActionDto);
 
         return VIEW_CLIENT_CYPHER_DETAIL;
     }
 
-    @PostMapping(value = "cypher/verify/{id}")
-    public String verifyCodeword(@PathVariable("id") final Long id,
-                                 @Valid @ModelAttribute(FORM_OBJECT_NAME) final CodewordFormDTO codewordFormDto,
+    @PostMapping(value = "cypher/verify")
```

Comment:
Možná tady taky můžeš nechat ID v URL?
Ať je to stejně? Dává to smysl?

Reply:
muze byt